packages/edit-site/src/components/add-new-template/style.scss

```
@@ -177,5 +129,71 @@
 		align-items: center;
 		justify-content: center;
 		height: 100%;
+		position: absolute;
+		left: 50%;
```

Nit: to achieve _true_ horizontal centering, we need to shift this element left by half its size

```suggestion
		left: 50%;
		transform: translateX(-50%);
```